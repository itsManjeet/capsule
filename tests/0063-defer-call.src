add := fun (a, b) {
	defer { println("defer 1"); };
	return 10;
	defer { println("defer 2"); };
};

defer { println("defer 3"); };
x := add (2, 4);
println (x);

defer { println("defer 4"); };