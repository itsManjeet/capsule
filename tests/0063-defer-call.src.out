defer 1
10
defer 4
defer 3
